docs/nl/docs/environment-variables.md

```diff
@@ -0,0 +1,298 @@
+# Omgevingsvariabelen
+
+/// tip
+
+Als je al weet wat "omgevingsvariabelen" zijn en hoe je ze kunt gebruiken, kun je deze stap gerust overslaan.
+
+///
+
+Een omgevingsvariabele (ook bekend als "**env var**") is een variabele die **buiten** de Python-code leeft, in het **besturingssysteem** en die door je Python-code (of door andere programma's) kan worden gelezen.
+
+Omgevingsvariabelen kunnen nuttig zijn voor het verwerken van applicatie **instellingen**, als onderdeel van de **installatie** van Python, enz.
+
+## Maak en gebruik omgevingsvariabelen
```

Comment:
```suggestion
## Omgevingsvariabelen maken en gebruiken
```
(this might be personal preference, but I thought the original titles read funny)